src/rust/en.mangairo/res/source.json

```
@@ -0,0 +1,25 @@
+{
+	"info": {
+		"id": "en.mangairo",
+		"lang": "en",
+		"name": "MangaIro",
+		"version": 1,
+		"url": "https://w.mangairo.com/home",
```

In order for deep links to work and such this should be the base url, even if it isn't the home listing page
```suggestion
		"url": "https://w.mangairo.com",
```